prov/efa/src/rdm/rxr_pkt_type_misc.c

```diff
@@ -76,11 +95,21 @@ ssize_t rxr_pkt_init_handshake(struct rxr_ep *ep,
 	 * Always include connid at the end of a handshake packet.
 	 * If peer cannot make use of connid, the connid will be ignored.
 	 */
-	connid_hdr = (struct rxr_handshake_opt_connid_hdr *)(pkt_entry->wiredata + pkt_entry->pkt_size);
+	connid_hdr = (struct rxr_handshake_opt_connid_hdr *)rxr_pkt_end(pkt_entry);
```

Comment:
I do not think use `rxr_pkt_end` is useful. it is better to use `pkt_entry->wiredata + pkt_entry->pkt_size)` because `pkt_entry->pkt_size` keep changing.

Reply:
Actually I find this pretty handy. There is already a `rxr_pkt_start` function and I was able to use `rxr_pkt_end` in a couple places to make the code more concise.

Reply:
Overall I think the name `rxr_pkt_start/end` is too ambiguous to be useful, because each packet entry has two parts: local info and wire data. The two parts are in two different buffers. So name `rxr_pkt_start/end` does not tell which part it refer to. So if we really need a utility function, the name need to be `rxr_pkt_entry_wiredata_star/end`.

The usage of the function here is even more confusing because usually `xxx_end` was applied on a constant object. but here  the size keep increasing.

Reply:
Thanks for clarifying. I do see convenience of these utilities. Renamed the function as suggested.